src/webapi/events/mouse.rs

```diff
@@ -72,6 +72,28 @@ pub trait IMouseEvent: IUiEvent {
         ).try_into().unwrap()
     }
 
+    /// Returns the X position on the target element where this event occured.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/offsetX)
+    // Not in w3c standard
```

Comment:
Please add links to the specs. (: 

While it's not in the official standard (yet?) there is a draft spec, e.g. here https://drafts.csswg.org/cssom-view/#ref-for-dom-mouseevent-offsetx